RoomControlService.cs

```diff
@@ -18,6 +18,7 @@ public static void AddingRoom(int RoomNo, int ID)
             using (var connection = new SqlConnection(_connectionString))
             {
                 connection.Open();
+                string insertRoom = "Insert into Room(ID, roomNo, roomColor, checkIn) values ( @ID, @roomNo, 'Salmon', GETDATE())";
```

Comment:
HowManyDay daha öncesinde  room' un veri tablosunda ekliydi. fakat artık bunun sadece Customer tablosuna ait bir veri olduğunu düşündüğüm için bu sorguda ve veri tablosundan kaldırdım. 
sizce nasıl olmuş? @zencifil